aspnet/tutorials/first-web-api.rst

```diff
@@ -201,30 +226,52 @@ The result appears in the sessions list. The response code should be 200. Use th
 Implement the other CRUD operations
 ------------------------------------
 
-The last step is to add ``Create``, ``Update``, and ``Delete`` methods to the controller. These methods are variations on a theme, so I'll just show the code and highlight the main differences.
+We'll add ``Create``, ``Update``, and ``Delete`` methods to the controller. These are variations on a theme, so I'll just show the code and highlight the main differences.
 
 Create
 ^^^^^^
 
 .. literalinclude:: first-web-api/sample/src/TodoApi/Controllers/TodoController.cs
   :language: c#
-  :lines: 33-42
+  :start-after: #region snippet_Create
+  :end-before: #endregion
   :dedent: 8
 
 This is an HTTP POST method, indicated by the `[HttpPost] <https://docs.asp.net/projects/api/en/latest/autoapi/Microsoft/AspNetCore/Mvc/HttpPostAttribute/index.html>`_ attribute. The `[FromBody] <https://docs.asp.net/projects/api/en/latest/autoapi/Microsoft/AspNetCore/Mvc/FromBodyAttribute/index.html>`_ attribute tells MVC to get the value of the to-do item from the body of the HTTP request.
 
 The `CreatedAtRoute <https://docs.asp.net/projects/api/en/latest/autoapi/Microsoft/AspNetCore/Mvc/Controller/index.html>`_ method returns a 201 response, which is the standard response for an HTTP POST method that creates a new resource on the server. ``CreateAtRoute`` also adds a Location header to the response. The Location header specifies the URI of the newly created to-do item. See `10.2.2 201 Created <http://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html>`_.
 
-We can use Fiddler to send a Create request:
+Using use Postman to send a Create request
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. image:: first-web-api/_static/pmc.png
+
+- Set the HTTP method to ``POST``
+- Set the type to JSON
```

Comment:
You have to select Body first, and when I do that I don't have the JSON dropdown to the right of the "binary" radio button in the location you show it.  Your screen shot shows tabs for GetAll, Create, etc., which the tutorial hasn't had the reader create.